ui/components/NewPlayer.jsx

```diff
@@ -91,10 +116,19 @@ export default class NewPlayer extends React.Component {
       );
     }
 
+    if (CustomNewPlayer) {
+      return (
+        <CustomNewPlayer
+          {...this.props}
+          handleNewPlayer={this.handleNewPlayer}
+        />
+      );
+    }
+
     return (
       <Centered>
```

Comment:
Looking good! I was confused at first, but this makes sense. 👍 
I think we should pull out this default form into its own component, so we can point to it as an example implementation for people who want to override with a custom new player component.

Reply:
Thank you. I agreed, i already extract the form into a single component.